core/base/argo-workflows/send-webhooks.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
```

Comment:
This seems to be used for only advsier 
how about we move this in adviser as well.

Reply:
I put it here because it is fairly generic. It can be used by other workflows if they have a similar need.

Reply:
sounds good :+1: 
thanks